sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,186 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/copy-spopersonalsitepage
+applicable: SharePoint Online
+title: Copy-SPOPersonalSitePage
+schema: 2.0.0
+author: xuyangzou
+ms.author: xuyangzou, spodeanu
+ms.reviewer:
+---
+
+# Copy-SPOPersonalSitePage
+
+## SYNOPSIS
+
+**What does this command do?**
```

Comment:
are these headers within SYNOPSIS allowed? I haven't seen other docs with such. I don't know the answer, hopefully someone from AskAdmin can review and confirm

Reply:
I actually think the SYNOPSIS part should be very brief -- a line or 2; then all the detailed info can be moved to the DESCRIPTION section.